lib/issuer.js

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const jsigs = require('jsonld-signatures');
+const {Ed25519Signature2018} = jsigs.suites;
+const vc = require('vc-js');
+
+// load config defaults
+require('./config');
+
+const walletClient = require('./walletClient');
+
+const instances = exports.instances = require('./instances');
+exports.users = require('./users');
+//exports.configurations = require('./configurations');
+
+exports.issue = async ({actor, accountId, issuer, credential}) => {
+  const {instance} = await instances.get({issuer});
+  const capability = instance.capability.find(
+    c => c.referenceId === 'assertionMethod');
+
+  // TODO: should be able to auto parse key ID and type from capability
+  // within `controllerKey.getAsymmetricKey`
+  const {invocationTarget} = capability;
+  const controllerKey = await walletClient.getControllerKey(
+    {actor, accountId});
+  const issuerKey = await controllerKey.getAsymmetricKey({
+    id: invocationTarget.id,
+    type: invocationTarget.type,
+    capability
+  });
+
+  // create vcSigner API
+  const vcSigner = {
+    // TODO: need to be able to get the public key ID associated with the
```

Comment:
@dlongley can you turn this into an issue?  Option in github to create issue directly from this comment.

Reply:
Filed #10.